### DISCUSSION TITLE
Add an easier way to search through open tabs

### DISCUSSION THREAD

**Discussion by ficd0** [zen-browser/desktop#2002] [Ideas]
One frustration I'm currently having with Zen's vertical tabs (and browsers in general), is that I need to parse through a lot of open tabs to find the one I want to focus. This is especially hard in vertical tabs in compact mode. I know there is a search shortcut to search through tabs, `%`, but this key is inconvenient to reach on most keyboards and it doesn't feel very smooth to search for tabs in this way. It is also not possible to change the search shortcut for tabs in the Search settings menu.

I recommend a configurable keybind that automatically focuses the search bar AND enters search in "Tabs" mode. I think this would go a long way towards QOL improvement for users that are generally more keyboard centric.

**Comment by SheepDomination** [comment]
Identical to that of Edge ? Edge has alot of unique features that are designed fairly well from a U.I perspective.

**Reply by ficd0** [reply]
I've never used edge myself so I wouldn't really know. However, I've heard that Edge is a good browser from a UX perspective (Microsoft spyware notwithstanding). I actually implemented a (admittedly very hacky) solution to this on my own system by putting the following in my i3 config: `bindsym --release $mod+t exec --no-startup-id xdotool key --clearmodifiers ctrl+l shift+5 space`. It's basically just a keybind that triggers a macro that presses `Ctrl+L`, `%` and `<space>` in sequence. While it's been working for me, I would definitely still love to see this feature supported by the browser itself, especially because needing to configure a system-wide macro just to perform one action in a single application is... not very efficient.

**Comment by grantith** [comment]
It may not be possible but a possible solution is allowing priority configurations for Firefox Suggest.
